Baskerville/llama2-inference-srun.md

```diff
@@ -0,0 +1,344 @@
+# Llama 2 inference on Baskerville using `srun`
+
+Before you can make use of them you’ll need to download the models.
+See [llama-download.md](llama-download.md) for how to go about doing this on Baskerville.
+
+# Llama 2 7B inference, single node, single GPU
```

Comment:
This is probably more a personal preference than anything but some markdown linters and, I presume, parsers will complain if there is more than one top level (single-#) heading in a doc so you may (or may not) want to move all remaining headings down one level with an extra #.

Reply:
Thanks; that's a good point. I've applied your suggestion and moved everything down a level.